### DISCUSSION TITLE
About the future of this projet

### DISCUSSION THREAD

**Discussion by Ilshidur** [Ilshidur/action-slack#255] [Announcements]
Hello everyone.

First of all I would like to thank all the people who _still_ use this GitHub Action and [every people who contributed to this project](https://github.com/Ilshidur/action-slack/graphs/contributors). 🙏

This project has been made public 4 years ago. This is with a little bit of shame that I would also like to apologize for not providing enough maintenance over the last 2 years, despite the various opened issues and PRs. As I stopped using Slack personally, I did not find any motivation to pursue maintenance on this project, nor did I think about writing a maintenance status in the readme. This was a selfish behavior and I am sorry about it.

Concerning what's coming next for this project, I consider stopping its maintenance, or at least deprecating it.
As Slack released [it's official GitHub Action](https://github.com/marketplace/actions/slack-send), I don't find any interest over maintaining an unofficial alternative, because I believe that the personal time I would commit to a project I don't use anymore will not be worth enough against the work of the Slack developers. I highly support open source and I believe that the official Action is **the** project to pursue. I don't plan to work on this project anymore.

If someone is willing to take over this project, or just be a maintainer, I would gladly discuss this. Admitting this happens, I will make sure to ensure the existing users of this project remain safe security-wise, as this situation can eventually raise security concerns. Also feel free to fork it. :)

**TL;DR : Looking for someone to eventually take over this project, as I am considering deprecating it in favor of the [official Slack Action](https://github.com/marketplace/actions/slack-send).**
